file_paths.md

```diff
@@ -253,6 +254,7 @@ create a `puppet` user or group.
                 pxp-access.log                # not enabled by default
             spool *                           # directory containing results of pxp-agent modules
             run *
+        tasks-cache *                         # directory containing a cache of files for all downloaded tasks
```

Comment:
I realize @MikaelSmith asked you to change this in a prior comment... but I'm wondering what went into that decision?